include/psen_scan_v2/monitoring_frame_msg.h

```diff
@@ -84,16 +86,17 @@ class MonitoringFrameMsg
   uint32_t scan_counter_{ 0 };
   std::vector<double> measures_;
   std::vector<double> intensities_;
-  std::vector<MonitoringFrameDiagnosticMessage> diagnostic_messages_;
+  std::vector<monitoring_frame::diagnostic::Message> diagnostic_messages_;
   bool diagnostic_data_enabled_{ false };
 
 public:
-  friend DynamicSizeRawData serialize(const MonitoringFrameMsg& frame);
-  friend MonitoringFrameMsg deserializeMonitoringFrame(const MaxSizeRawData& data, const std::size_t& num_bytes);
+  friend DynamicSizeRawData serialize(const Message& frame);
```

Comment:
The code, like here, is inconsistent. Sometimes `Message` and sometimes `monitoring_frame::Message` is used. I suggest to use only one version, preferably, with regard to readability, the following version: 
```suggestion
  friend DynamicSizeRawData serialize(const monitoring_frame::Message& frame);
```